src/Label/Label.less

```diff
@@ -39,92 +44,30 @@
 }
 
 .Label--default {
-  .colorize(@neutral_silver);
+  background-color: @neutral_silver;
 }
 
 .Label--new-feature {
-  .colorize(@accent_teal_tint_2);
+  background-color: @label_purple;
 }
 
 .Label--success {
-  .colorize(@alert_green_tint_2);
+  background-color: @label_green;
 }
 
 .Label--warning {
-  .colorize(@alert_orange_tint_2);
+  background-color: @label_yellow;
 }
 
 .Label--error {
-  .colorize(@alert_red_tint_2);
-}
-
-.Label--category-1 {
-  .colorize(@accent_purple_tint_2);
+  background-color: @label_red;
 }
 
 .Label--category-2 {
```

Comment:
unfortunate naming but this is still used in prod (I think just in apps dash) so I can't get rid of it, but design also didn't provide this as an option technically in the official dewey figma I think https://www.figma.com/file/IBVNvo2dI221k4Dye4Hg0S/Dewey-2.0-WIP?node-id=21222%3A943

Reply:
Good to know! We aren't using `label--category-1` anywhere though. Is that correct?

Reply:
We are using the color option CATEGORY_1 but not the class. In Label.tsx we map colors to classes, so CATEGORY_1 now maps to new-feature which is the purple one, so it'll stay purple (see the Label.tsx file)